frontend/components/BottomNav.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import {bottomStates} from './States';
+import { FAB } from 'react-native-paper';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+export interface Props {
+    currState: string;
+    numItems?: number;
+    totalPrice?: number;
+}
+
+export class BottomNav extends Component {
```

Comment:
Would you be interested in re-writing the class components as function components? With the introduction of Hooks, function components are looking like the future of React, so it would be good to understand them, but if it's too much of a hassle, we can worry about that later